datafusion/physical-expr/src/aggregate/min_max.rs

```diff
@@ -835,6 +889,44 @@ impl AggregateExpr for Min {
         )))
     }
 
+    fn groups_accumulator_supported(&self) -> bool {
+        Max::groups_accumulator_supported(&Max::new(
+            self.expr.clone(),
+            self.name.clone(),
+            self.data_type.clone(),
+        ))
```

Comment:
I wonder why this implementation is different than `self.data_type.is_primitive()` of Max::groups_accumulator_supported.